src/common/dsdl/ocvsmd/common/svc/node/UavcanNodeExecCmdReq.0.1.dsdl

```diff
@@ -0,0 +1,78 @@
+# This is copy/paste of the request part of the `uavcan/node/435.ExecuteCommand.1.3.dsdl` service definition.
```

Comment:
These are copy/paste from corresponding public DSDL types - unfortunately it's impossible (currently) to reference request or receive part of a service inside another dsdl type. @pavel-kirienko said it could be relatively easy to add such support in the future. It it happens then we will drop these copies.

Reply:
I think this change won't even affect Nunavut at all; it can be confined to PyDSDL only, as the final AST will look the same.